common/remoteviewinterface.h

```diff
@@ -76,6 +76,8 @@ public slots:
 
     virtual void setViewActive(bool active) = 0;
 
+    virtual void setUpdateInterval(int ms) = 0;
```

Comment:
This is not something anyone externally should control, it's purely an implementation detail to reduce event loop load. Can we get away without having this somehow?